contrib/vcloud-benchmark.py

```diff
@@ -46,12 +48,21 @@ def download_required_jars(config):
         cmd += ["-warn"]
     cmd += ["-retrieve", "lib/vcloud-jars/[artifact](-[classifier]).[ext]"]
 
-    # install vcloud jar and dependencies
-    subprocess.run(
-        cmd,
-        cwd=_ROOT_DIR,
-        shell=vcloudutil.is_windows(),  # noqa: S602
-    )
+    # Provide temporary directory
+    if config.freshIvyCache:
+        temp_dir = tempfile.mkdtemp()
+        cmd += ["-cache", str(temp_dir)]
+        logging.debug(f"Using fresh ivy cache: {temp_dir}")
```

Comment:
Is that log message relevant? The directory will soon be deleted anyway.

Furthermore, for log statements please do not use f-strings, pass the parameters individually in order to allow lazy string formatting.